src/app/shared/utils/consts.ts

```diff
@@ -57,5 +57,34 @@ export const Consts = {
         'azure-file' : 'Azure File Service'
     },
     S3_HOST_IP: '',
-    S3_HOST_PORT: ''
+    S3_HOST_PORT: '',
+    API: {
+        AUTH : {
+            'getToken' : 'identity/v3/auth/tokens'
+        },
+        DELFIN : {
+            'storages' : 'delfin/storages',
+            'storagePools' : 'delfin/storage-pools',
+            'volumes' : 'delfin/volumes'
+        },
+        HOTPOT : {
```

Comment:
Why HOTPOT API calls added here?

Reply:
This will require checks in multiple files and that will need to be done in a different PR. I have started adding the APIs to the Consts file starting with Delfin APIs. 
There is an issue to move all API endpoints to a constants file #84 . I will reference that when i fix this. 
This is not in the scope of the current PR.

Reply:
I would suggest, let's not then mix it here. Just in case someone refers this PR in future, they should not get confused.

Reply:
Fixed. Removed the references to non Delfin API endpoints. Pushed in the latest commit.